pandas/io/stata.py

```diff
@@ -3402,6 +3512,11 @@ class StataWriterUTF8(StataWriter117):
 
         .. versionadded:: 1.1.0
 
+    value_labels : dict of dicts
+        Dictionary containing columns as keys and dictionaries of column value
+        to labels as values. Labels for a single variable must be 32,000
```

Comment:
Is this the total length of all labels for a single variable mus tbe 32000 or smaller, or each label must be 32k or smaller?  Small clarification would be useful.